File: perl/src/tests/t_init_in_advance.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$f = Atrshmlog::get_init_buffers_in_advance();

print "init in advance : $f : \n";

$old = Atrshmlog::set_init_buffers_in_advance_on();

print "init in advance : $old : \n";

$f = Atrshmlog::get_init_buffers_in_advance();

print "init in advance : $f : \n";

$old = Atrshmlog::set_init_buffers_in_advance_off();

print "init in advance : $old : \n";

$f = Atrshmlog::get_init_buffers_in_advance();

print "init in advance : $f : \n";

$old = Atrshmlog::set_init_buffers_in_advance_on();

print "init in advance : $old : \n";

$f = Atrshmlog::get_init_buffers_in_advance();

print "init in advance : $f : \n";


print " \n";

exit (0);

# end of main

File: python/src/tests/t_init_in_advance.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

f = atrshmlog.get_init_buffers_in_advance()

print('init in advance : ' + str(f) + ' : ')

old = atrshmlog.set_init_buffers_in_advance_on()

print('init in advance : ' + str(old) + ' : ')

f = atrshmlog.get_init_buffers_in_advance()

print('init in advance : ' + str(f) + ' : ')

old = atrshmlog.set_init_buffers_in_advance_off()

print('init in advance : ' + str(old) + ' : ')

f = atrshmlog.get_init_buffers_in_advance()

print('init in advance : ' + str(f) + ' : ')

old = atrshmlog.set_init_buffers_in_advance_on()

print('init in advance : ' + str(old) + ' : ')

f = atrshmlog.get_init_buffers_in_advance()

print('init in advance : ' + str(f) + ' : ')


print (' ')

exit(0)

# end of test

